packages/@angular/cli/commands/test.ts

```diff
@@ -14,13 +14,15 @@ export interface TestOptions {
   build?: boolean;
   sourcemap?: boolean;
   progress?: boolean;
+  config: string;
 }
 
 
 const TestCommand = EmberTestCommand.extend({
   availableOptions: [
     { name: 'watch', type: Boolean, default: true, aliases: ['w'] },
     { name: 'code-coverage', type: Boolean, default: false, aliases: ['cc'] },
+    { name: 'config', type: String, aliases: ['c', 'cf'] },
```

Comment:
I think it's ok enough. It's true that `-c -cc` is kinda weird, but I'd expect `-c` to be used much more than `-cc`, and less so them together. And when users use them together, they shouldn't have to change `-c` to `-cf`. 

Looking at the PoV of making mistakes, since before both `-c` and `-cf` were allowed then you could just as easily mistakenly do `-cc` instead if you were used to `-c`.